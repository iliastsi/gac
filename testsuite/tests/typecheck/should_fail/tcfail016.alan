param_def() : proc
  foo(i : int) : proc
    i : byte[10];
  {}
{}
