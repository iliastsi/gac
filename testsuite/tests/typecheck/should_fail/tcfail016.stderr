tcfail016.alan:3:5: Error: Conflicting definitions for `i'
	Bound at: tcfail016.alan:2:7
	          tcfail016.alan:3:5

